keras/preprocessing/image.py

```diff
@@ -152,12 +146,24 @@ def __init__(self,
                  height_shift_range=0.,
                  shear_range=0.,
                  horizontal_flip=False,
-                 vertical_flip=False):
+                 vertical_flip=False,
+                 dim_ordering='th'):
         self.__dict__.update(locals())
         self.mean = None
         self.std = None
         self.principal_components = None
         self.lock = threading.Lock()
+        assert dim_ordering in {'tf', 'th'}, 'dim_ordering must be in {tf, th}'
```

Comment:
Good point.  Does the exception in the updated commit match what you were thinking.